src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -411,6 +460,12 @@ public WebElement getDisplayedToStudentCheckBox(int instrNum) {
                                                          + "']"));
     }
 
+    public WebElement getDisplayNameField(int instrNum) {
+        return browser.driver.findElement(By.cssSelector("#instructorTable" + instrNum + " input[name='"
+                + Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME
+                + "']"));
```

Comment:
Extract the string or the expression `By.cssSelector(...)` as a variable for better readability.